tools/mo/unit_tests/moc_tf_fe/check_info_messages.py

```diff
@@ -46,7 +46,6 @@ def arg_parse_helper(input_model,
         source_layout={},
         target_layout={},
         freeze_placeholder_with_value=None,
-        tensorflow_use_custom_operations_config=None,
         data_type=None,
         tensorflow_custom_operations_config_update=None,
```

Comment:
What is our plan about this option `tensorflow_custom_operations_config_update` and is the difference?
@slyalin, @jane-intel, could you please comment on it?

Reply:
This does not block this PR to proceed